sklearn/datasets/openml.py

```diff
@@ -424,17 +424,34 @@ def _get_data_qualities(data_id, data_home):
         return None
 
 
-def _get_data_shape(data_qualities):
-    # Using the data_info dictionary from _get_data_info_by_name to extract
-    # the number of samples / features
+def _get_data_instances(data_qualities):
```

Comment:
```suggestion
def _get_num_samples(data_qualities):
```
I think this would be clearer naming for readers of scikit-learn code.

Reply:
I changed the name, thanks.